algorithms/spot_finding/factory.py

```diff
@@ -61,7 +61,7 @@ def generate_phil_scope():
                 "to be accepted by the filtering algorithm."
         .type = int(value_min=1)
 
-      max_spot_size = 100
+      max_spot_size = 1000
```

Comment:
Slightly worried that this might have a negative effect on the standard spotfinding?

Reply:
max_spot_size=100 is overly conservative, especially in the context of smaller eiger pixels (remember this is the number of spot pixels in 3 dimensions).

Reply:
And I suppose rings picked up will either be noise or eliminated by the other filters.